specs/experimental/plasma.md

```diff
@@ -51,6 +52,18 @@ Any DA provider can implement the following endpoints to receive and serve input
     200 OK
   ```
 
```

Comment:
Can you elaborate on why there are 2 ways to do it, if both most be implemented, why one would be implemented over the other

Reply:
Will do. The first way is backwards compat with S3 DA server, the second is because with blockchain like DA, you can't know the commitment ahead of time